bin/MessagesBundle_it.properties

```diff
@@ -53,14 +53,14 @@ Circ_opt = Opzioni
 Macro_origin = Mostra origine per macro
 View = Vista
 Export_completed = Esportazione riuscita
-Export_error = Errore durante l'esportazione: 
+Export_error = Errore durante l'esportazione:
 PDF_Font_error = Certi font non sono stati esportati. Utilizzare Helvetica, Courier, Times or Symbol.
 Illegal_filename = Nome del file non valido.
 File_not_found = File non trovato.
 Print_uncomplete = Impossibile terminare la stampa.
-Open_error = Errore di accesso al file: 
-Save_error = Errore durante il salvataggio:  
-Time = Tempo impiegato 
+Open_error = Errore di accesso al file:
```

Comment:
Same here, check if the space is required for the spacing. If not, it is OK to remove it.